packages/builder-css-styles/box-shadow/index.ts

```diff
@@ -2,12 +2,17 @@ import { BoxShadowManifest } from "@reflect-ui/core/lib/box-shadow";
 import { color } from "../color";
 import { px } from "../dimensions";
 
-export function boxshadow(bs: BoxShadowManifest): string {
-  if (!bs) {
+export function boxshadow(bs: BoxShadowManifest[]): string {
```

Comment:
```ts
function boxshadow (...bs: BoxShadowManifest[]){
    /// ...
}
```

Reply:
Why use Spread Operator to objectify bs?